### DISCUSSION TITLE
Serialiastion Formats

### DISCUSSION THREAD

**Discussion by ghost** [FamilySearch/GEDCOM#129] [General]
A question came up in a recent webinar about other serialisation formats, e.g. XML, JSON, and so on.

I understood that the rules for such formats would eventually appear as appendices or other adjuncts to the GEDCOM spec. Is this planned during V7's development?

Clearly we need those rules where general-purposes formats such as XML and JSON are concerned because the actual encoding is open to many variations, and it must be guaranteed that conversion between any of the supported formats is mechanical and loss-free.

**Comment by tychonievich** [comment]
Discussed in the steering committee meeting 2022-04-19. Separating the data model from the serialization format and adding other serialization formats is not one of the steering committee's short-term objectives.

In principle, a third party could define a simple structure-to-element transformation, such as could be implemented in an XML-to-GEDCOM XSLT, without needing any changes to the specification. We'd welcome suggestions on this from the community if anyone is interested in working on this.
